code/game/dna/mutations/monkey_mutation.dm

```diff
@@ -26,7 +26,22 @@
 	H.invisibility = 101
 	var/has_primitive_form = H.dna.species.primitive_form // cache this
 	if(has_primitive_form)
-		H.set_species(has_primitive_form, keep_missing_bodyparts = TRUE)
+
+		if(ispath(has_primitive_form, /mob/living/simple_animal/nian_caterpillar))
+			var/datum/mind/M = H.mind
+			//addtimer(CALLBACK(src, PROC_REF(become_headslug), user), 5 SECONDS)
```

Comment:
is there any real reason for these lines to be commented out, rather than removed?

Reply:
No, they can be removed.